checkout-core/src/main/java/com/adyen/checkout/core/ui/model/ExpiryDate.kt

```diff
@@ -3,13 +3,10 @@
  *
  * This file is open source and available under the MIT license. See the LICENSE file for more info.
  *
- * Created by ararat on 17/7/2024.
+ * Created by ozgur on 4/10/2024.
  */
-package com.adyen.checkout.ui.core.internal.ui.model
+package com.adyen.checkout.core.ui.model
 
-import androidx.annotation.RestrictTo
-
-@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 data class ExpiryDate(
```

Comment:
Should `EMPTY_DATE` and `INVALID_DATE` be made internal/restricted? Maybe they could be even moved to an internal class so that merchants don't see/use them. I see they are used inside the validator but I don't think the public validator should depend on them

Reply:
Good one, updated the Validator